src/components/organisms/Header.tsx

```diff
@@ -97,7 +98,7 @@ export default function Header() {
           <Heading as="h1">Sablier Docs</Heading>
         </Title>
         <Subtitle>
-          <p>Documentation for the Sablier token distribution protocol</p>
+          <p>Documentation for Sablier Lockup and Sablier Flow</p>
```

Comment:
I will undo this change because Sablier token distribution protocol = Lockup and Flow